packages/waku/src/router/define-router.ts

```diff
@@ -367,6 +369,7 @@ export function unstable_defineRouter(fns: {
             type: 'file',
             pathname,
             body: handleApi(pathname, {
+              url: new URL(pathname, 'http://localhost:3000'),
```

Comment:
Should this always be localhost:3000?

Is the port configurable?

Reply:
It shouldn't matter. They shouldn't rely on it. It can be without the port.
```suggestion
              url: new URL(pathname, 'http://localhost'),
```

Reply:
Or, we could make `url` optional. Do you prefer that?

Reply:
hmm i'll look as I implement the remaining parts